meorg_client/client.py

```diff
@@ -4,11 +4,13 @@
 import json
 import os
 from typing import Union
+from urllib.parse import urljoin
+from meorg_client.exceptions import RequestException
+
+VALID_METHODS = ["POST", "GET", "DELETE", "PUT"]
```

Comment:
consider moving const out of array and then create array from those const
```
POST = "POST"
GET = "GET"
VALID_METHODS = [POST, GET]
```
This way const can be used used independently as well

Reply:
Moved valid methods to new constants.py file coming in next commit. A variable that points to a string of the same characters is not going to save any code here and they are only used a handful of times in the single client class.